docs/README.md

```diff
@@ -21,6 +21,7 @@ The benefit to using Semantic Conventions is in following a common naming scheme
 Semantic Conventions are defined for the following areas:
 
 * **[General](general/README.md): General Semantic Conventions**.
+* [CICD](cicd/cicd-metrics.md): Semantic Conventions for CICD systems.
```

Comment:
Later when we have additional docs under `cicd/` for spans as well we can create a `cicd/README.md` and link to that instead.